k0459866/Assignments/assignment02.py

```diff
@@ -0,0 +1,17 @@
+week = 7
+# days in a week
+
+weekends = 2
+# counts both days in a standard weekend
```

Comment:
I didn't mean to go back and change older things, I mainly copy and pasted them from github because I made a mistake and deleted my Assignments folder instead of moving the files before deleting. If you'd like, I'll go back and edit all of my past files to make them correct.